packages/@lwc/engine-server/src/__tests__/fixtures/scoped-slots/for-each/expected.html

```diff
@@ -6,18 +6,14 @@
     <x-child-with-for-each>
       <ul>
         <li>
-          ‍‍
```

Comment:
iirc, there is a special handling on SSR and hydration logic to handle these. should we remove that?